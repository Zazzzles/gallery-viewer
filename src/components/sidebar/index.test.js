import React from 'react';
import renderer from 'react-test-renderer';

import Sidebar from './index';
import NavigationContext from '../../context/navigation-context';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <NavigationContext.Provider value={{ xIndex: 0, yIndex: 0 }}>
        <Sidebar collections={[]} />
      </NavigationContext.Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
