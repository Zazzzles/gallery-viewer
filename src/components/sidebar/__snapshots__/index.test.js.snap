// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Container"
>
  <div
    className="LogoWrapper"
  >
    <img
      alt="Main logo"
      className="Logo"
      src="main-logo.png"
    />
  </div>
  <div
    className="MenuItemWrapper"
  >
    <div
      className="MenuItemWrapperInner"
      style={
        Object {
          "transform": "translate3d(0, -0px, 0)",
        }
      }
    />
  </div>
</div>
`;
